.github/workflows/check-drone-signature.md

```diff
@@ -0,0 +1,60 @@
+# Check Drone Signature
+
+## Overview
+
+This document describes the intention and usage of the `check-drone-signature` GitHub Actions workflow.
+You can find [the workflow file at `.github/workflows/check-drone-signature.yml`][wf-file].
```

Comment:
This bit seems redundant since it's in the filename and if you followed a link it'd be in the link name, but keeping the ref to the workflow itself might be useful. 🙂